public/data/ste001.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Stefano Merazzi",
+  "type": "personal",
+  "bio": "Web Developer | Mobile Developer",
+  "avatar": "https://github.com/ste001.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ste001",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/FearTheSock",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/stefano-merazzi",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Gitlab",
```

Comment:
I recommend capitalizing the `l` here. What do you think?
```suggestion
      "name": "Follow me on GitLab",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Yeah, works better since the actual name is "GitLab". I'll update ASAP

Reply:
Done